src/main/java/com/example/ifclubserver/post/domain/entity/Post.java

```diff
@@ -29,6 +30,10 @@ public class Post {
     @Column(name = "updated_at", nullable = false)
     private LocalDateTime updatedAt;
 
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "member_id", nullable = false)
+    private Member member;
+
     @Builder
     public Post(String title, String content, LocalDateTime createdAt, LocalDateTime updatedAt) {
```

Comment:
```suggestion
    public Post(String title, String content, LocalDateTime createdAt, LocalDateTime updatedAt, Member member) {
```
테이블간의 연관관계를 맺고 나면, 실제 Post 엔티티를 생성할 때에도 Member 객체를 넣어서 생성해주어야 할 것 같아요..!